hammer_cli_foreman_bootdisk.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = Dir['LICENSE', 'README*']
   s.require_paths = ["lib"]
 
-  s.add_dependency 'hammer_cli_foreman', '>= 0.1.2'
```

Comment:
Please, don't pin hammer core version like that. We mostly use "minimal" version without any upper limit. In worst cases we limit up to next major version, e.g. 4.0.0.